config/jobs/cert-manager/istio-csr/cert-manager-istio-csr-presubmits.yaml

```diff
@@ -282,3 +282,57 @@ presubmits:
         options:
           - name: ndots
             value: "1"
+
+  # kind based istio-csr e2e job for Kubernetes v1.21, istio v1.11
+  - name: pull-cert-manager-istio-csr-k8s-v1-21-istio-v1-11
+    context: pull-cert-manager-istio-csr-k8s-v1-21-istio-v1-11
+    # Match everything except PRs that only touch docs/
```

Comment:
I think we don't actually skip the docs changes in this job? (should be a `skip_if_only_changed` field https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md#triggering-jobs-based-on-changes)
Same with the other presubmit

Reply:
Thanks, I think I want to just always run these jobs for all PRs. I will remove the false comments.